src/reducers/statusEffects.js

```diff
@@ -1,21 +1,26 @@
 import { BLEED_OUT } from '../actions/bleeding';
 import { FEEL_TIRED } from '../actions/tired';
 
+const singleStackEffects = [BLEED_OUT, FEEL_TIRED];
+const isSingleStackEffect = singleStackEffects.includes;
+
+const multiStackEffects = [];
```

Comment:
None of these yet, but added just to show what the logic might look like if there were some.